### PR TITLE
Update fabricbot config to handle issue triaging

### PR DESCRIPTION
**Pull request by Evangelink**
cc @MarcoRossignoli @jakubch1 @pavelhorak

### CODE REVIEW

File: .github/fabricbot.json [hunk 1]
```diff
@@ -1,6 +1,413 @@
 {
   "version": "1.0",
   "tasks": [
+    {
+      "taskType": "trigger",
+      "capabilityId": "IssueResponder",
+      "subCapability": "IssuesOnlyResponder",
+      "version": "1.0",
+      "config": {
+        "taskName": "Add needs triage label to new issues",
```
Comment (Evangelink) [inline .github/fabricbot.json:10]: Automatically adds `needs-triage` label to unassigned new issues.

File: .github/fabricbot.json [hunk 2]
```diff
@@ -1,6 +1,413 @@
...
+        ]
+      }
+    },
+    {
+      "taskType": "trigger",
+      "capabilityId": "IssueResponder",
+      "subCapability": "IssueCommentResponder",
+      "version": "1.0",
+      "config": {
+        "taskName": "Replace needs author feedback label with needs attention label when the author comments on an issue",
```
Comment (Evangelink) [inline .github/fabricbot.json:61]: Task 2

File: .github/fabricbot.json [hunk 3]
```diff
@@ -1,6 +1,413 @@
...
+        ]
+      }
+    },
+    {
+      "taskType": "trigger",
+      "capabilityId": "IssueResponder",
+      "subCapability": "IssuesOnlyResponder",
+      "version": "1.0",
+      "config": {
+        "taskName": "Remove no recent activity label from issues",
```
Comment (Evangelink) [inline .github/fabricbot.json:117]: Task 3

File: .github/fabricbot.json [hunk 4]
```diff
@@ -1,6 +1,413 @@
...
+        ]
+      }
+    },
+    {
+      "taskType": "trigger",
+      "capabilityId": "IssueResponder",
+      "subCapability": "IssueCommentResponder",
+      "version": "1.0",
+      "config": {
+        "taskName": "Remove no recent activity label when an issue is commented on",
```
Comment (Evangelink) [inline .github/fabricbot.json:161]: Task 4

File: .github/fabricbot.json [hunk 5]
```diff
@@ -1,6 +1,413 @@
...
+        ]
+      }
+    },
+    {
+      "taskType": "scheduled",
+      "capabilityId": "ScheduledSearch",
+      "subCapability": "ScheduledSearch",
+      "version": "1.1",
+      "config": {
+        "taskName": "Add no recent activity label to issues",
```
Comment (Evangelink) [inline .github/fabricbot.json:193]: Task 5

File: .github/fabricbot.json [hunk 6]
```diff
@@ -1,6 +1,413 @@
...
+        ]
+      }
+    },
+    {
+      "taskType": "scheduled",
+      "capabilityId": "ScheduledSearch",
+      "subCapability": "ScheduledSearch",
+      "version": "1.1",
+      "config": {
+        "taskName": "Close duplicate issues",
```
Comment (Evangelink) [inline .github/fabricbot.json:309]: Task 6

**Review by jakubch1 — COMMENTED** [review]
LGTM

**Pull request merged**